cmd/cnbBuild.go

```diff
@@ -37,6 +37,13 @@ type cnbBuildUtilsBundle struct {
 	*docker.Client
 }
 
+type cnbBuildTelemetryData struct {
+	Version               int             `json:"version"`
+	Config                cnbBuildOptions `json:"config"`
+	ProjectDescriptorUsed bool            `json:"projectDescriptorUsed"`
```

Comment:
Wouldn't we want to get the actual content of the project descriptor, rather than just if a descriptor has been provided?

Reply:
Really? Should we store content of a project we do not have even access to? Feels strange...

Reply:
I guess we cannot do this.